src/soa/mormot.soa.core.pas

```diff
@@ -689,6 +689,9 @@   TServiceContainer = class(TInterfaceResolverInjected)
     /// retrieve all registered Services contracts as a JSON array
     // - i.e. a JSON array of TServiceFactory.Contract JSON objects
     function AsJson: RawJson;
+    // delete previously registered Interfaces

```

Comment:
please use 3 / for the first comment line ///
(for proper documentation generation)

Reply:
Done.